bot/cogs/moderation/moderation_commands.py

```
@@ -0,0 +1,76 @@
+
+        await member.ban(reason=f"For violating discord server rules.")
+        await ctx.send(f"{member.mention} has been ban in this server for good.")
+
+
+    @commands.hybrid_command(name="unban", description="Unban a user")
+    @commands.has_permissions(manage_guild=True)
+    @guard("unban")
+    async def unban(self, ctx: commands.Context, member: discord.Member):
+        settings = await self.get_mod_settings(str(ctx.guild.id))
+        if not settings.is_moderation_settings_enabled:
+            await ctx.send(f"⚠️ Moderation system is disabled in this server.")
```

Ruff (0.14.6)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)